cudatoolkit-dev/scripts/cuda_activate.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash 
+
+export PATH_BACKUP=$PATH
+export CUDA_HOME_BACKUP=$CUDA_HOME
+export CUDA_HOME=$CONDA_PREFIX/pkgs/cuda
+export PATH=$CUDA_HOME/bin:$PATH
```

Comment:
Rather than setting and unsetting environment variables, we should probably soft link (`ln -s`) on activate and remove links on deactivate.